cfme/tests/services/test_provision_stack.py

```diff
@@ -197,21 +36,18 @@
 @pytest.yield_fixture(scope="function")
 def template(provider, provisioning, dialog_name):
     template_type = provisioning['stack_provisioning']['template_type']
-    if provider.type == 'azure':
-        template_name = 'azure-single-vm-from-user-image'
-    else:
-        template_name = fauxfactory.gen_alphanumeric()
-
+    template_name = fauxfactory.gen_alphanumeric()
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=template_name)
 
     if provider.type == "ec2":
```

Comment:
If all the data file names match the provider type strings, you can condense this into a single call that parses in the provider type `'{}_vm_template.data'.format(provider.type)`